### PR TITLE
Fix light preview parameters

### LINKED ISSUE #2981: Temporary GUI light has different angle than when it is added to the scene
With #2947 merged, it is now even more apparent that the temporary spot light used in GUI when interactively adding lights has wrong "shining angle":

https://user-images.githubusercontent.com/182533/115684549-bdfeb280-a357-11eb-880f-90fbee9006a2.mp4

https://user-images.githubusercontent.com/182533/115684562-bfc87600-a357-11eb-96fa-c6f096f0b351.mp4

As can be seen in the videos, the wrong size now affects both the light source and the visual (i.e. the visual's size is correct regarding the temporary light's size).

Or is it just so that the temporary light has different default properties than the default added light?

### PR DESCRIPTION
**Pull request by peci1**
Fixes #2981 . See the issue for detailed description.

https://user-images.githubusercontent.com/182533/115697965-661a7880-a364-11eb-9ac8-74f06fbdc843.mp4

I also wanted to add tests for the inner/outer angle, but they have no getters. Should I add these? Please, also check that the copy of `light` I make in the test is actually still a pointer to the preview light and is not pointing to the same object `light` does at the end of the test. I think it should be so, but another pair of eyes would be good :)

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]
Thank you for the fix, 

Regarding the inner/outer angles tests, Adding new methods don't break API, I think it's fine to add them, @chapulina or @scpeters Do you agree?

File: gazebo/rendering/Light.cc [hunk 1]
```diff
@@ -156,14 +156,20 @@ void Light::UpdateSDFFromMsg(const msgs::Light &_msg)
 //////////////////////////////////////////////////
 void Light::UpdateFromMsg(ConstLightPtr &_msg)
 {
-  this->UpdateSDFFromMsg(*_msg);
+  this->UpdateFromMsg(*_msg);
```
Comment (scpeters) [inline gazebo/rendering/Light.cc]: I think this `_msg` pointer should be checked before dereferencing it

**Review by scpeters — COMMENTED** [review]
> Regarding the inner/outer angles tests, Adding new methods don't break API, I think it's fine to add them, @chapulina or @scpeters Do you agree?

you are welcome to add getters for inner and outer angles if you like

File: gazebo/rendering/Light.cc (hunk 1 above)
Comment (peci1) [reply gazebo/rendering/Light.cc]: Resolved (it ignores null messages).

**Comment by peci1** [comment]
I added the getters and added a test for spotlight creation which utilizes the getters.

**Review by chapulina — APPROVED** [review]

**Comment by chapulina** [comment]
New test passed on CI, merging!

**Pull request merged**